core-bundle/src/EventListener/PreviewUrlConvertListener.php

```diff
@@ -16,6 +16,7 @@
 use Contao\CoreBundle\Event\PreviewUrlConvertEvent;
 use Contao\CoreBundle\Framework\ContaoFramework;
 use Contao\PageModel;
+use Symfony\Component\HttpFoundation\Request;
```

Comment:
Are these changes related in any way or is that just CS? If there are changes then the tests should reflect those and otherwise I'm confused why this is in this PR (not saying you have to do another one but for next time...I just don't know what this is doing here).

Reply:
They are, see `private function getParams(Request $request): ?string`: https://github.com/contao/contao/pull/1286/files#diff-3ce6d6db43287783796b3f1f09f1f81fR62

Reply:
I think he means all changes in this file. For me most of it looks like CS or rather restructuring, but it seems that `$article->id` got changed to `$article->alias`.

Reply:
The PR does not include CS fixes. I have fixed and refactored the `__invoke()` method though; it used `$article->id` although it should have used `$article->alias` to be in line with the `Controller::redirectToFrontendPage()` method:

https://github.com/contao/contao/blob/68b47c6ccab103c8fd205bc1c9a7f105c293d86b/core-bundle/src/Resources/contao/library/Contao/Controller.php#L1359-L1363

Reply:
What I‘m saying is that if it‘s CS, it doesn‘t belong here. If it‘s a bugfix, tests need to reflect the changes😊 So apparently the tests are missing which might cause a future regression.

Reply:
Unit test added in 7dfad9c2f022a5fb50a28f8d3b90f833bc68137b.